src/rail/estimation/algos/deepdisc.py

```diff
@@ -310,7 +313,7 @@ class DeepDiscPDFEstimator(CatEstimator):
     """DeepDISC estimator"""
 
     name = "DeepDiscPDFEstimator"
-    config_options = CatInformer.config_options.copy()
+    config_options = {}
```

Comment:
This isn't an `informer` stage, so we shouldn't use the CatInformer configs. But the `CatEstimator` configs include an `hdf5_groupname` value by default, and that messes up our file caching algorithm. 

Two other options:
* Use `RailStage.config_options.copy()` to get the options from the next level up. 
* Use `CatEstimator.config_options.copy()` and confirm that we can override the `hdf5_groupname` StageParam with something like None?

Reply:
Looks like the first option is better.  Setting `hdf5_groupname=None` with CatEstimator options still gives an error